index.html

```diff
@@ -147,36 +194,25 @@ <h2>Profesional Art Printing Data</h2>
                     <button class="big-button" type="button">See Project</button>
                 </div>  
 
-                <div class="grid-item">
-                    <div class="multi-post2">
```

Comment:
- [ ] As written in the project requirements. I kindly suggest that you ensure that when a project is hovered on with the mouse:
-  the description and dark opacity disappear
-  and a See Project button appears to the bottom of the image and takes the whole image width.
-   Your `See Project` button should link to the Live Demo of the project or to the GitHub repository.

- [ ] [OPTIONAL] Please ensure that your padding for your project is similar to the template design. Kindly see the images below

| Your Design  |Template Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/52934128/178112591-2aecddbf-19ab-4b28-a88f-dd7f7596991d.png) | ![image](https://user-images.githubusercontent.com/52934128/178112550-f1b33dc5-838e-4251-898a-ea299dc20431.png) |